portfolio/src/main/webapp/index.html

```diff
@@ -116,13 +116,13 @@ <h2>Comments</h2>
        </div>
     </div>
 
-     <script src="https://www.gstatic.com/charts/loader.js"></script>
-        <script>
-        google.charts.load('current', {packages: ['corechart']});
-        google.charts.setOnLoadCallback(loadChart);
-        </script>
+    <script src="https://www.gstatic.com/charts/loader.js"></script>
+    <script>
+    google.charts.load('current', {packages: ['corechart']});
+    google.charts.setOnLoadCallback(getPhrase);
+    </script>
 
-        <div id="activityPieChart"/>
+    <div id="commentBarChart"/>
     </div>
     </div>
```

Comment:
Where are the open div tags for these closed ones?

Reply:
Oh, they have no open div tags, I'll delete them